packages/client/tests/functional/_utils/getTestSuitePlan.ts

```diff
@@ -40,6 +40,10 @@ export function getTestSuitePlan(
     return getExpandedTestSuitePlanWithProviderFlavors(config)
   })
 
+  expandedSuiteConfigs.forEach((config) => {
+    config.matrixOptions.engineType ??= testCliMeta.engineType
```

Comment:
Makes this information available in the test suite itself (in case we need to skip tests for any reason)